.travis.yml

```diff
@@ -45,4 +46,5 @@ matrix:
       env: DB=oracle
   allow_failures:
     - php: hhvm
+      env: DB=oracle  # until it is fixed
```

Comment:
missing `-` before this:
```yml
    - php: hhvm
    - env: DB=oracle
```

Otherwise it allows failure on hhvm `&&` oracle, instead of `||`